merlin-sdk/src/test/java/gov/nasa/jpl/aerie/merlin/protocol/types/SerializedValueTest.java

```diff
@@ -0,0 +1,26 @@
+package gov.nasa.jpl.aerie.merlin.protocol.types;
+
+import java.util.List;
+import java.util.Map;
+
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public final class SerializedValueTest {
+  @Test
+  public void testHashCode() {
```

Comment:
We shouldn't need to test that Java's own `hashCode` methods abide by its own contract 🤔

Reply:
Just removed this test :)